frontend/public/style/mixin/_scroll-shadows.scss

```diff
@@ -0,0 +1,38 @@
+$scroll-shadows-bg-color: rgba(255, 255, 255, 1);
+$scroll-shadows-bg-color-transparent: rgba(255, 255, 255, 0);
+
```

Comment:
Comment `Horizontal scroll drop shadows` and `@mixin scroll-shadows-horizontal` should probably match?  It might help avoid future drift.

Reply:
Agreed.  Favor updating the comment instead of changing the mixin.